safe_transaction_service/history/models.py

```diff
@@ -1646,10 +1647,25 @@ def matching_for_address(self, address: str):
         return self.filter(Q(address=address) | Q(address=None))
 
 
+def _validate_webhook_url(url: str) -> None:
```

Comment:
Maybe we can refactor this method and tests at some point to `gnosis-py`, maybe even a new `ModelField` integrating this serializer and the `max_lenght` set to 255 by default. Just an idea for the future